etna/loggers/base.py

```diff
@@ -0,0 +1,192 @@
+from abc import ABC
+from abc import abstractmethod
+from typing import Any
+from typing import Dict
+from typing import Iterable
+from typing import Union
+
+import pandas as pd
+
+from etna.core.mixins import BaseMixin
+
+
+class Logger(ABC, BaseMixin):
```

Comment:
Maybe `Logger` -> `BaseLogger` (`AbstractLogger`)?
To explicit point that it is abstract class